docker/api/container.py

```diff
@@ -139,6 +139,12 @@ def create_host_config(self, *args, **kwargs):
         kwargs['version'] = self._version
         return utils.create_host_config(*args, **kwargs)
 
+    def create_networking_config(self, *args, **kwargs):
+        return utils.create_networking_config(*args, **kwargs)
+
+    def create_endpoint_config(self, *args, **kwargs):
+        return utils.create_endpoint_config(self._version, *args, **kwargs)
```

Comment:
`create_endpoint_config` needs to pass the version through, same as `create_host_config`. `create_networking_config` doesn't need to *right now*, but it might in the future, and I'd like to avoid having to do another deprecation like we did with `create_host_config`.